src/commands/synthetics/__tests__/reporters/default.test.ts

```diff
@@ -191,6 +191,7 @@ describe('Default reporter', () => {
   describe('runEnd', () => {
     beforeEach(() => {
       writeMock.mockClear()
+      reporter.reportStart({startTime: Date.now() - 567890}) // 9m 28s
```

Comment:
Just to be safe we can add a `jest.useFakeTimers()` (nitpicky though, I think the test will alway evaluate under a second)
```suggestion
      jest.useFakeTimers()
      reporter.reportStart({startTime: Date.now() - 567890}) // 9m 28s
```